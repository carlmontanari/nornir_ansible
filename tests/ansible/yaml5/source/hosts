all:
  children:
    iosxr:
      hosts:
        lab_a_router_01:
    eric_eccli:
      hosts:
        lab_a_router_02:
