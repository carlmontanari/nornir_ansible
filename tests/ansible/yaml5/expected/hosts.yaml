lab_a_router_02:
  connection_options: {}
  data: {}
  groups:
  - eric_eccli
  hostname: lab_a_router_02
  password: null
  platform: null
  port: null
  username: null
lab_a_router_01:
  connection_options: {}
  data: {}
  groups:
    - iosxr
  hostname: lab_a_router_01
  password: null
  platform: null
  port: null
  username: null